.github/workflows/release_finish.yaml

```diff
@@ -0,0 +1,122 @@
+---
+name: Release Finalize
+on:
+  push:
+    branches:
+      - main
+    paths:
+      # Only trigger for new releases
```

Comment:
I don't understand the connection between the comment and version.bzl file, could you explain it to me, please?

Reply:
This workflow takes release candidates and creates an actual release once the change has been merged to `main`. The expectation is that `version.bzl` would only change for a new release.

Reply:
Oh I think you overestimated my understanding of Github actions :)) But now I took another look in the morning and I think I get it now - lines 1 - 9 in this file tell Github actions to trigger this job every time there is a commit to main branch modifying version.bzl file. I just didn't get that after a long day :)